### DISCUSSION TITLE
What is the "type":"Follow" response format of inbox?

### DISCUSSION THREAD

**Discussion by springrain** [mastodon/mastodon#25011] [Q&A] [answered]
I use the go language to implement some functions. Mastodon’s Follow request can be obtained normally, but I don’t know what format the return value should be? Please help me, thank you very much

**Comment by trwnh** [comment, marked as answer]
Accept or Reject?

**Reply by springrain** [reply]
What is the JSON format that returns response? I didn't find it in the docs

**Comment by springrain** [comment]
I just learned that Accept or Reject is notified through the inbox parameter, rather than directly responding to the inbox request
